### DISCUSSION TITLE
STAC 2.1 Dev Branch

### DISCUSSION THREAD

**Discussion by Xylopyrographer** [Xylopyrographer/STAC#51] [Ideas]
@mqshaw 
How goes it Mark. All well I trust.

I've created a `2.1_dev branch` and pushed a new version of STAC that'd be great to get your comments and feedback on.

Main features:
* Ability to do OTA firmware updates.
* Added mDNS support.
* When operating in Peripheral Mode, the Tally Mode can now be selected as either Camera Operator or Talent.

More details in the NewFeatures.txt and the /Documentation/Detailed Change Log.md files.

This code is essentially the same as a version I've created for a STAC that drives a larger 8 by 8 matrix ( ~6.5 cm square). Need to design a case and finalize the hardware design for that yet. Doesn't use an ATOM MATRIX.

Please hit me up with any questions.

**Comment by mqshaw** [comment]
Hi Rob,

Thanks for the update.  I will pull the latest version, build it and get it loaded on our STACs so that we can start testing it.
Best,
Mark

From: Xylopyrographer ***@***.***>
Sent: Sunday, January 29, 2023 11:42 PM
To: Xylopyrographer/STAC ***@***.***>
Cc: Mark Shaw ***@***.***>; Mention ***@***.***>
Subject: [Xylopyrographer/STAC] STAC 2.1 Dev Branch (Discussion #51)


@mqshaw<https://github.com/mqshaw>
How goes it Mark. All well I trust.

I've created a 2.1_dev branch and pushed a new version of STAC that'd be great to get your comments and feedback on.

Main features:

  *   Ability to do OTA firmware updates.
  *   Added mDNS support.
  *   When operating in Peripheral Mode, the Tally Mode can now be selected as either Camera Operator or Talent.

More details in the NewFeatures.txt and the /Documentation/Detailed Change Log.md files.

This code is essentially the same as a version I've created for a STAC that drives a larger 8 by 8 matrix ( ~6.5 cm square). Need to design a case and finalize the hardware design for that yet. Doesn't use an ATOM MATRIX.

Please hit me up with any questions.

**Comment by Xylopyrographer** [comment]
@mqshaw 
Hope you've had a great week.

Just pushed what I think is the final code for v2.1.

All documentation has been revised where needed to match.

I'll go hands off for a week on the code. Let me know how it works for you!

Cheers.

**Comment by Xylopyrographer** [comment]
Version 2.1 released moments ago!
